### ISSUE TITLE
311 and Airport links on the global navigation are missing info for the screen reader 

### ISSUE THREAD

**Issue by kriskristin** [cityofaustin/techstack#3336]
**Acceptance criteria**
Probably `aria-label` should be `Airport — opens in new window` and `3-1-1 — opens in new window`

![image](https://user-images.githubusercontent.com/5839508/67970017-47587d00-fbd8-11e9-844c-f703b8779a12.png)

**Describe the bug**
Screen reader only tells the user that the 'Library' and '311' these are external links. It doesn't give the user any context of where the links go.
**[URL](https://janis.austintexas.io/en/health-safety/healthcare-prevention/disease-prevention/mobile-food-vendor-permit-guide/#Contact-information)**

**To Reproduce**
[See Video](https://drive.google.com/open?id=1HJmhoeeLQKzKo1RTzIkJAuBNcWvr1mPE)
User says: I don't know what these links are, it just says "opens a new window'

**Current behavior**
Screen reader only says 'external link'
 
**Expected behavior**
Screen reader should tell the user what the labels are and where they will go.

**Device**
Desktop

**Browser**
IE

**Comment by easherma** [comment]
https://janis-3336-aria-label-external.netlify.com

**Comment by desigonz** [comment]
@easherma 

![image](https://user-images.githubusercontent.com/5839508/68698472-8a0a4580-0546-11ea-8192-de3fb1ada18d.png)

Missing punctuation between label and "opens in new window." Reads funny on a screenreader without it :)

**Comment by easherma** [comment]
@desigonz should be fixed now!

**Comment by desigonz** [comment]
thanks! qa done
